src/plugin/regions.js

```diff
@@ -23,7 +23,9 @@ class Region {
                 : Number(params.end);
         this.resize =
             params.resize === undefined ? true : Boolean(params.resize);
+        this.isResizing = false;
         this.drag = params.drag === undefined ? true : Boolean(params.drag);
+        this.isDragging = false;
         this.loop = Boolean(params.loop);
```

Comment:
can you add some comment for both the isResizing and isDragging (here or elsewhere)?

Reply:
ok, i will do this. hope this solution works for the issue opener